site/go.mod

```diff
@@ -2,4 +2,4 @@ module sigs.k8s.io/kueue/site
 
 go 1.24
 
-require github.com/google/docsy v0.10.0 // indirect
+require github.com/google/docsy v0.11.0 // indirect
```

Comment:
Why can't the bot upgrade this one?

Reply:
We don't have a Dependabot configuration for the `./site/go.mod`. I'm not sure it's even possible to configure Dependabot for this, since updating dependencies requires using `hugo mod get github.com/google/docsy` and `hugo mod tidy`.

Reply:
Oh, I see. The `go mod tidy` does not seem to work well.
Alright, let's keep this one for now. Thank you